development/tools/cmd/synchronizer/main.go

```diff
@@ -0,0 +1,192 @@
+package main
```

Comment:
what should be exit code when we found that some component is out-dated? IMO 1, because then ProwJob should fail.

Reply:
IMO job should fail only if scripts return fatal, information about out-dated components are sent to slack